nominal:
  goal:
    coordinates:
      - 8.0
      - 4.0
      - 0.79
    padding:
      - 0.05
      - 0.05
      - 0.5
    max_time: 30.0
    time_intervals: 101
    solver_accuracy: "high"
  frequency: 20.0
cbf:
  gamma: 0.2
external:
  max_time_without_new_message: 1.0  # seconds
  max_time_without_modified_message: 5.0  # seconds
limits:
  max_vel: 1.0
  min_vel: 0.1
  max_omega: 0.2
  
